packages/openapi-generator/src/generator.ts

```diff
@@ -166,7 +183,7 @@ async function generateFromFile(
   try {
     openApiDocument = await convertOpenApiSpec(filePath);
   } catch (err) {
-    logger.error(
+    await logger.error(
```

Comment:
maybe leftover?

Reply:
yes, I found that the logger is not synchronous, but the await did also not help. Only console.error directly put the message out.